api/tests/routes/native/v2/bookings_test.py

```diff
@@ -0,0 +1,570 @@
+
+        client = client.with_token(booking.user.email)
+        response = client.get("/native/v1/bookings")
+
+        assert response.status_code == 200
+        assert response.json["ongoing_bookings"][0]["stock"]["offer"]["venue"]["name"] == "Public name"
+
+    def test_return_venue_banner_url(self, client):
+        venue = offerers_factories.VenueFactory(bannerUrl="http://bannerUrl.com")
+        booking = booking_factories.BookingFactory(stock__offer__venue=venue)
+
+        client = client.with_token(booking.user.email)
+        response = client.get("/native/v1/bookings")
+
+        assert response.status_code == 200
+        assert response.json["ongoing_bookings"][0]["stock"]["offer"]["venue"]["bannerUrl"] == "http://bannerUrl.com"
+
+    def test_return_venue_is_open_to_public(self, client):
+        venue = offerers_factories.VenueFactory(isOpenToPublic=True)
+        booking = booking_factories.BookingFactory(stock__offer__venue=venue)
+
+        client = client.with_token(booking.user.email)
+        response = client.get("/native/v1/bookings")
+
+        assert response.status_code == 200
+        assert response.json["ongoing_bookings"][0]["stock"]["offer"]["venue"]["isOpenToPublic"] is True
+
+
+class GetBookingTicketTest:
+    identifier = "pascal.ture@example.com"
+
+    def test_get_booking_withdrawal_informations(self, client):
+        user = users_factories.BeneficiaryGrant18Factory(email=self.identifier)
+        booking_factories.BookingFactory(
+            user=user,
+            stock__offer__subcategoryId=subcategories.CONCERT.id,
+            stock__offer__withdrawalDetails="Veuillez chercher votre billet au guichet",
+            stock__offer__withdrawalType=offer_models.WithdrawalTypeEnum.ON_SITE,
+            stock__offer__withdrawalDelay=60 * 30,
+        )
+
+        with assert_num_queries(2):  # user + booking
+            response = client.with_token(self.identifier).get("/native/v2/bookings")
+            assert response.status_code == 200
+
+        ticket = response.json["ongoing_bookings"][0]["ticket"]
+        assert ticket["withdrawal"]["details"] == "Veuillez chercher votre billet au guichet"
+        assert ticket["withdrawal"]["type"] == "on_site"
+        assert ticket["withdrawal"]["delay"] == 60 * 30
+
+    def test_get_booking_no_ticket(self, client):
+        user = users_factories.BeneficiaryGrant18Factory(email=self.identifier)
+        booking_factories.BookingFactory(
+            user=user,
+            stock__offer__subcategoryId=subcategories.CONCERT.id,
+            stock__offer__withdrawalType=offer_models.WithdrawalTypeEnum.NO_TICKET,
+        )
+
+        with assert_num_queries(2):  # user + booking
+            response = client.with_token(self.identifier).get("/native/v2/bookings")
+            assert response.status_code == 200
+
+        ticket = response.json["ongoing_bookings"][0]["ticket"]
+        assert ticket["noTicket"] == True
```

Comment:
```suggestion
        assert ticket["noTicket"] is True
```
Y en d'autres plus bas aussi, pour des `False` et des `None`